client/app/controllers/project.js

```diff
@@ -0,0 +1,10 @@
+import Controller from '@ember/controller';
+import { optionset } from '../helpers/optionset';
+
+export default class ProjectController extends Controller {
+  get publicStatusNotNullVisibilityGeneralPublic() {
```

Comment:
very long name 😅  
Maybe something just like `publicStatusGeneralPublicProjects()` ?
Also including `Projects1 in the name makes clear what this getter returns

Reply:
sounds good -- I'll shorten it :)

Reply:
Yeah, I've been seeing a little bit of this — the naming of things should abstract away what the code is doing, not really describe the code itself (in camelcase).

Reply:
Another thing I like to keep in mind when I'm naming stuff (and this comes from Sandi Metz) is that if my method can be named with a conjunction "_and_", it probably needs to be split up.